src/main/java/org/ow2/proactive/addons/email/EmailSender.java

```diff
@@ -73,6 +79,10 @@
 
     public static final String ARG_BODY = "body";
 
+    public static final String ARG_FILETOATTACHE = "file_path";
```

Comment:
ARG_FILETOATTACHE => ARG_FILETOATTACH please change this in other parts of the code